salt/states/flatpak.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+"""
+Management of flatpak packages
+==============================
+Allows the installation and uninstallation of flatpak packages.
+
+.. versionadded:: Neon
+"""
+from __future__ import absolute_import, print_function, unicode_literals
+
+import salt.utils.path
+
+__virtualname__ = 'flatpak'
+
+
+def __virtual__():
+    if salt.utils.path.which('flatpak'):
+        return __virtualname__
+
+    return False, 'The flatpak state module cannot be loaded: the "flatpak" binary is not in the path.'
+
+
+def installed(location, name):
+    """Ensure that the named package is installed.
+
+    Args:
+        location (str): The location or remote to install the flatpak from.
+        name (str): The name of the package or runtime.
+
+    Returns:
+        dict: The "result" and "output".
+    """
```

Comment:
Instead of a CLI Example, we need an example here. What would a state file look like that uses this state.

```
Example

.. code-block:: bash

    install_package:
      flatpack.installed:
        - location: <whatever goes here>
        - name: gimp or whatever
```